cli/cli-client/src/main/java/org/infinispan/cli/connection/rest/RestConnection.java

```diff
@@ -511,7 +511,19 @@ public String execute(List<CommandInputLine> commands) throws IOException {
                }
                break;
             }
-            case Stats.CMD:
+            case Stats.CMD: {
+               Resource resource = activeResource;
+               if (command.hasArg(CliCommand.NAME)) {
+                  resource = pathToResource(command.arg(CliCommand.NAME));
+               }
+               if (resource instanceof CacheResource) {
+                  response = client.cache(resource.getName()).stats();
+               } else if (resource instanceof ContainerResource) {
+                  response = client.cacheManager(resource.getName()).stats();
+               } else {
+                  throw MSG.invalidResource(resource.getName());
```

Comment:
nitpick, when I run `stats` in the root, the error message is `Invalid resource ''`. 
I would suggest replacing the empty string with `/`

Reply:
Done